core/src/main/java/org/apache/iceberg/deletes/BaseDVFileWriter.java

```diff
@@ -143,7 +143,7 @@ private void write(PuffinWriter writer, Deletes deletes) {
 
   private PuffinWriter newWriter() {
     EncryptedOutputFile outputFile = fileFactory.newOutputFile();
-    String ident = "Iceberg " + IcebergBuild.fullVersion();
+    String ident = IcebergBuild.fullVersion();
```

Comment:
nit: you might want to just inline it: `return Puffin.write(outputFile).createdBy(IcebergBuild.fullVersion()).build();`